.ci/renewable-deploy/configuration-replacer.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -ex
+
+# Create real parameters files to be modified and applied to the cluster as configmaps
+cp app/config/domains_urls.yml.dist app/config/domains_urls.yml
+cp app/config/parameters.yml.dist app/config/parameters.yml
+
+DOCKER_PHP_FPM_IMAGE=${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG_PHP_FPM}
+DOCKER_ELASTIC_IMAGE=${DOCKER_USERNAME}/elasticsearch:${DOCKER_IMAGE_TAG_ELASTIC}
+PATH_CONFIG_DIRECTORY='app/config'
+GOOGLE_CLOUD_STORAGE_BUCKET_NAME=''
+
+FILES=$( find kubernetes -type f )
+VARS=(
+    FIRST_DOMAIN_HOSTNAME
+    SECOND_DOMAIN_HOSTNAME
+    THIRD_DOMAIN_HOSTNAME
+    DOCKER_PHP_FPM_IMAGE
+    DOCKER_ELASTIC_IMAGE
+    PATH_CONFIG_DIRECTORY
+    GOOGLE_CLOUD_STORAGE_BUCKET_NAME
+    GOOGLE_CLOUD_PROJECT_ID
+    GKE_LOAD_BALANCER_IP
+)
+
+for FILE in $FILES; do
+    for VAR in ${VARS[@]}; do
+        sed -i "s|{{$VAR}}|${!VAR}|g" "$FILE"
+    done
+done
+unset FILES
+unset VARS
+
+# Set domain urls
+yq write --inplace app/config/domains_urls.yml domains_urls[0].url https://${FIRST_DOMAIN_HOSTNAME}
+yq write --inplace app/config/domains_urls.yml domains_urls[1].url https://${SECOND_DOMAIN_HOSTNAME}
+yq write --inplace app/config/domains_urls.yml domains_urls[2].url https://${THIRD_DOMAIN_HOSTNAME}
+# Add a mask for trusted proxies so that load balanced traffic is trusted and headers from outside of the network are not lost
+yq write --inplace app/config/parameters.yml parameters.trusted_proxies[+] 10.0.0.0/8
+
+# create links for additional kbuernetes configuration
+mkdir -p kubernetes/base/config/docker/postgres
+ln -f docker/postgres/postgres.conf kubernetes/base/config/docker/postgres/postgres.conf
+mkdir -p kubernetes/kustomize/overlays/production/config/docker/nginx/google-cloud
+ln -f docker/nginx/google-cloud/nginx.conf kubernetes/kustomize/overlays/production/config/docker/nginx/google-cloud/nginx.conf
+mkdir -p kubernetes/base/config/app/config
+ln -f app/config/domains_urls.yml kubernetes/base/config/app/config/domains_urls.yml
+ln -f app/config/parameters.yml kubernetes/base/config/app/config/parameters.yml
```

Comment:
removed all symlinks in https://github.com/shopsys/demoshop/pull/84/commits/970f9fe4b72cf478e1cc60c0a5b25ce99c3866ba 🚀

Reply:
Actually I'm taking this back. Moving `domains_urls.yml` and `parameters.yml` into kubernetes directory is not possible as this directory is not builded into image.
Maybe it could be done by setting the volumeMounts in a better way. Do you have any suggestions?

Reply:
Right now we keep it like it is and we try to solve the problem in shopsys/shopsys#1277